build.gradle.kts

```diff
@@ -60,6 +61,28 @@ kotlin {
         binaries.executable()
     }
 
+    // According to https://kotlinlang.org/docs/native-target-support.html
+    // Tier 1
+    macosX64()
+    macosArm64()
+    iosSimulatorArm64()
+    iosX64()
+
+    // Tier 2
+    // linuxX64()  // Commenting it out for a moment because its tests fail, and I cannot reproduce it locally.
```

Comment:
@aSemy this target seems to be special, the tests fail for it: https://github.com/charleskorn/kaml/actions/runs/8534050619/job/23377766881. Do you have any intuition why? I cannot reproduce it locally. My next idea is to configure Gradle build scans so that we see the whole stack trace.

Reply:
Next step: check what part of the assertion fails. "kotlin.AssertionError at **null:-1"** looks weird.